pkg/webhooks/resource/handlers.go

```diff
@@ -121,6 +124,7 @@ func (h *handlers) Validate(ctx context.Context, logger logr.Logger, request *ad
 	if err != nil {
 		return errorResponse(logger, err, "failed create policy context")
 	}
+	policyContext.InformerCacheResolvers = h.informerCacheResolvers
```

Comment:
Move this in the policy context builder.

Reply:
fixed